pilot/pkg/serviceregistry/kube/controller/multicluster.go

```diff
@@ -110,23 +131,32 @@ func (m *Multicluster) AddMemberCluster(clientset kubernetes.Interface, metadata
 
 	_ = kubectl.AppendServiceHandler(func(svc *model.Service, ev model.Event) { m.updateHandler(svc) })
 	_ = kubectl.AppendInstanceHandler(func(si *model.ServiceInstance, ev model.Event) { m.updateHandler(si.Service) })
+
 	go kubectl.Run(stopCh)
 	opts := Options{
 		ResyncPeriod: m.ResyncPeriod,
 		DomainSuffix: m.DomainSuffix,
 	}
+	webhookConfigName := strings.ReplaceAll(validationWebhookConfigNameTemplate, validationWebhookConfigNameTemplateVar, m.secretNamespace)
 	if m.fetchCaRoot != nil {
 		nc := NewNamespaceController(m.fetchCaRoot, opts, clientset)
 		go nc.Run(stopCh)
+		go util.PatchCertLoop(features.InjectionWebhookConfigName.Get(), webhookName, m.caBundlePath, clientset, stopCh)
+		valicationWebhookController := util.CreateValidationWebhookController(clientset, dynamicClient, webhookConfigName,
```

Comment:
We'd better not do this in the utils, which should include only functions.

Same applies to `PatchCertLoop`.

I think they maybe ` pkg/webhooks` is a good place for them

Reply:
agree, think we can address this is a sperate PR?

Reply:
I think this is to be backported to 1.6, right? If so, please address it in this one?

Reply:
to get into 1.6, need to be part of master first :)